mod-source-record-manager-server/src/main/java/org/folio/dao/util/TenantUtil.java

```diff
@@ -0,0 +1,40 @@
+package org.folio.dao.util;
+
+import io.vertx.core.Future;
+import io.vertx.core.Promise;
+import io.vertx.core.Vertx;
+import io.vertx.sqlclient.Row;
+import io.vertx.sqlclient.RowSet;
+import org.folio.rest.persist.PostgresClient;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+/* The util class helps to work with tenant's data */
+public class TenantUtil {
+
+  private TenantUtil() {
+  }
+
+
+  /* The method returns all the available tenants for whom the mode is deployed */
+  public static Future<List<String>> getModuleTenants(Vertx vertx) {
```

Comment:
It should not be utility class, it should be common spring service class, that has vertx as param in constructor, working on this change

Reply:
Moved from utility class to spring based component